app/scripts/services/deployments.js

```diff
@@ -367,7 +367,7 @@ angular.module("openshiftConsole")
       return scalableName === deployment.metadata.name;
     };
 
-    DeploymentsService.prototype.groupByService = function(/* deployments or deployment configs */ resources, services) {
+    DeploymentsService.prototype.groupByService = function(resources, services) {
```

Comment:
add a comment that its resources that replicate pods and have a selector defined